pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -70,6 +68,14 @@ func (e ShortcutExpander) getAll() []schema.GroupResource {
 	return availableAll
 }
 
+func (e ShortcutExpander) getServerResources() ([]*metav1.APIResourceList, error) {
+	if e.discoveryClient == nil {
```

Comment:
nice ! I like the idea. Yeah, seems like `ShortcutExpander` is used only within `utils` pkg.